include/lbann/utils/protobuf/decl.hpp

```diff
@@ -205,6 +205,20 @@ std::string serialize(google::protobuf::Message const& msg);
 void serialize(std::string const& pbuf_filename,
                google::protobuf::Message const& msg);
 
+///@}
+/** @name Writing protobufs */
+///@{
+
+/** @brief Assign a range of values to a repeated protobuf field. */
+template <typename T, typename ContainerT>
+void assign_to_repeated(google::protobuf::RepeatedField<T>& field,
+                        ContainerT const& values);
+/** @brief Concatenate a container of strings into a single string.
+ *  Elements are space separated.
+ */
+template <typename ContainerT>
+std::string to_space_sep_string(ContainerT const& values);
```

Comment:
Please add testing for these functions to [the Protobuf utilities tests](https://github.com/LLNL/lbann/blob/develop/src/utils/unit_test/protobuf_utils_test.cpp).